src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupBlackListReadFilter.java

```diff
@@ -39,6 +36,11 @@
     // Command line parser requires a no-arg constructor
```

Comment:
Could you update the comment on blacklistEntries to describe what it's storing.  It took me longer than it should have to understand a what this class was doing and why it wasn't just a list of read group names.

Reply:
The argument doc could also use a bit more info to point out that it can take slightly more complex inputs than just a name.  Maybe "The name of the read group to filter out or read group filter expression"

Reply:
Yeah, good point. I had to derive that when I looked at it too. Done.